### DISCUSSION TITLE
我用支付宝支付成功，没有升级成专业用户，请帮忙处理。

### DISCUSSION THREAD

**Discussion by i-vyou** [getlantern/lantern-forum-cn#892] [General]
我在这里完成的支付：https://buy.stripe.com/14k7v577O0Qq4s8eV3  ，支付的订单号见下图，可能我支付的地方不正确，请帮忙处理。
![ld](https://github.com/user-attachments/assets/0ae4222d-5cd2-4265-8bd5-5fdc060dc8e7)

**Comment by i-vyou** [comment]
我的账号邮箱：mgwms@sina.com

**Comment by MaxMKMax** [comment]
@i-vyou 非常抱歉. 好的. 我已经帮您提交上去了. 请您放心! 谢谢您支持蓝灯!

**Reply by i-vyou** [reply]
@Mike777Khoo 请帮忙跟踪一下问题，谢谢！

**Comment by MaxMKMax** [comment]
@i-vyou @i-vyou 好的. 帮您追问了. 刚好负责人有点事. 会尽快给您处理. 请您放心. 谢谢您对我们的信任!

**Reply by bubulei** [reply]
你好，我也完成支付宝支付，也没升级成专业版，账号是：769132445@qq.com

**Reply by i-vyou** [reply]
@Mike777Khoo 问题还未解决啊！

**Reply by MaxMKMax** [reply]
@i-vyou 您好. 我还等待着回复我呢. 负责人最近请病假呢. 不好意思.

**Comment by MaxMKMax** [comment]
@bubulei 您好. 请您提供我支付单据.

**Comment by greySkiesMarsh** [comment]
@i-vyou Hi there, I'm sorry for the delay in getting to this issue. I'm showing your Pro account is set to expire on 08/07/2027. Is that what it shows on your device?

**Comment by i-vyou** [comment]
@greySkiesMarsh 始终显示是免费版本，反复卸载重新安装还是这个问题，如图：
![ld](https://github.com/user-attachments/assets/85441c39-d213-42d8-9bce-15d1ce2656fc)

**Comment by MaxMKMax** [comment]
@i-vyou 你好。请您看一下这个方法。

1. 从文件资源管理器中，导航至 Windows(C:) 驱动器
2. 选择“用户”文件夹
3. 选择与安装 Lantern 的帐户相对应的 Windows 用户文件夹。
4. 从功能区菜单中选择“查看”按钮，将鼠标悬停在“显示”选项上，然后选择“隐藏项目”。您现在应该看到一个名为“AppData”的文件夹。
5. 打开“AppData”文件夹。
6.删除`Lantern`文件夹。
7. 下载并重新安装最新版本的 Lantern，并为 Pro 重新授权您的设备。

**Comment by MaxMKMax** [comment]
@bubulei 您好. 请您提供我支付单据. 这样我才能帮您提交问题上去

**Reply by bubulei** [reply]
![bbd55fe644ed86d2af9f2c2e357f4db](https://github.com/user-attachments/assets/4e2ece24-20c4-42fe-8da6-5d0d6088175d)
账号是：769132445@qq.com

**Comment by i-vyou** [comment]
@Mike777Khoo，@greySkiesMarsh 我的问题还未解决啊！能否发个激活码呢？以上方法都试过了。

**Reply by lushiyi23** [reply]
感覺好多人都是這個問題...不會要跑路了吧好擔心

**Comment by i-vyou** [comment]
@Mike777Khoo ，@greySkiesMarsh 我的问题还未解决啊！能否发个激活码呢？以上方法都试过了,还是不行啊！！
